plugin/src/main/kotlin/net/octyl/ts2kt/gradle/repository/npm/NpmClientRepository.kt

```diff
@@ -62,8 +64,22 @@ class NpmClientRepository(private val project: Project) : ClientRepository {
     private val registryUrl: String by lazy {
         val cap = ByteArrayOutputStream()
 
+        val npm = with (OperatingSystem.current()) {
+            when {
+                // on Windows, NPM is a .bat file that needs to run under cmd.exe
+                isWindows -> arrayOf("cmd", "/c", "npm")
```

Comment:
Is this really better than just running `npm.bat`? Or `npm.exe`?

Reply:
It's actually required to run. There's no shebang line in batch files, so you have to run the parent process (cmd) and specify the script file as an argument.